provisioner/shell-local/provisioner.go

```diff
@@ -2,14 +2,14 @@ package shell
 
 import (
 	"context"
-
 	"github.com/hashicorp/hcl/v2/hcldec"
 	sl "github.com/hashicorp/packer/common/shell-local"
 	"github.com/hashicorp/packer/packer"
 )
 
 type Provisioner struct {
-	config sl.Config
+	config      sl.Config
+	isValidated bool
```

Comment:
Hm I feel a bit bad that a provisioner had to know about that and would recommend against doing this 🤔